src/main/scala/com/wavesplatform/network/RxExtensionLoader.scala

```diff
@@ -178,7 +177,7 @@ object RxExtensionLoader extends ScorexLogging {
         case ApplierState.Applying(maybeBuffer, applying) =>
           if (applying != extension) log.warn(s"Applied $extension doesn't match expected $applying")
           maybeBuffer match {
-            case None => syncNext(state.copy(applierState = ApplierState.Idle))
+            case None => state.copy(applierState = ApplierState.Idle)
```

Comment:
Why not sync next?

Reply:
because the local score comes to RxExtensionLoader after an extension applying, so `syncNext` will happen anyway. An error during an extension applying is handled by `onExtensionApplied`